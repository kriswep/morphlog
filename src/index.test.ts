import { GraphQLServer } from 'graphql-yoga';

import { server } from './index';

jest.mock('graphql-yoga', () => ({
  GraphQLServer: function(props) {
    expect(props).toMatchSnapshot();
    this.mockServerStart = jest.fn();
    return { start: () => true };
  },
}));

test('index should start GraphQLServer', async () => {
  expect(server).toBeDefined();
});
