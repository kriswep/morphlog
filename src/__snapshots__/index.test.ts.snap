// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
Object {
  "context": [Function],
  "resolverValidationOptions": Object {
    "requireResolversForResolveType": false,
  },
  "resolvers": Object {
    "AuthPayload": Object {
      "user": [Function],
    },
    "Mutation": Object {
      "addChange": [Function],
      "addProjectMember": [Function],
      "addTeamMember": [Function],
      "createProject": [Function],
      "createTeam": [Function],
      "login": [Function],
      "signup": [Function],
      "updateChange": [Function],
    },
    "Query": Object {
      "change": [Function],
      "changes": [Function],
      "me": [Function],
      "project": [Function],
      "projects": [Function],
      "team": [Function],
      "teams": [Function],
    },
  },
  "typeDefs": "./src/schema.graphql",
}
`;
